test/UnitTest/Microsoft.AspNet.OData.Test.Shared/HttpConfigurationExtensionsTest.cs

```diff
@@ -192,6 +192,19 @@ public void EnableContinueOnError_Sets_ContinueOnErrorKeyFlag()
             Assert.True(config.HasEnabledContinueOnErrorHeader());
         }
 
+        [Fact]
+        public void EnableODataCaseInsensitiveModelBinding_Sets_CaseInsensitiveModelBindingKeyFlag()
+        {
+            // Arrange
+            HttpConfiguration config = new HttpConfiguration();
+
+            // Act
+            config.EnableODataCaseInsensitiveModelBinding();
+
+            // Assert
+            Assert.True(config.HasEnabledCaseInsensitiveModelBinding());
```

Comment:
Assuming the default is `false`, you could check if case insensitive model binding flag is `false `before `config.EnableODataCaseInsensitiveModelBinding();` is executed.

Reply:
#Resolved